model-optimizer/extensions/ops/dft.py

```diff
@@ -0,0 +1,133 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from mo.front.common.partial_infer.utils import int64_array
+from mo.graph.graph import Node, Graph
+from mo.ops.op import Op
+
+
+class FFTBase(Op):
+    enabled = False
+    op = None
+    version = 'opset7'
+
+    def __init__(self, graph: Graph, attrs: dict):
+        mandatory_props = {
+            'out_ports_count': 1,
+            'in_ports_count': 3,
+            'version': self.version,
+            'infer': self.infer
+        }
+        super().__init__(graph, mandatory_props, attrs)
+
+    def infer(self, node: Node):
```

Comment:
There is MxNet 1D FFT operation:  https://mxnet.apache.org/versions/1.1.0/api/python/ndarray/contrib.html#mxnet.ndarray.contrib.fft. We should support it as well.

Reply:
We must read MxNet (I)FFT as separate operation (e.g., MxNetFFT) and use the special transformation to convert MxNetFFT to (I)DFT.

Reply:
Such operation, corresponding extractors, and the corresponding transformation are written.